docker/bin/stop-presto

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This works for worker only. Coordinator doesn't support graceful shutdown.
+# This script will block until the server has actually shutdown.
+set -x
+
+http_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.http.port' | sed 's/^.*=\(.*\)$/\1/')"
+https_port="$(cat /usr/lib/presto/etc/config.properties | grep 'http-server.https.port' | sed 's/^.*=\(.*\)$/\1/')"
```

Comment:
A line can be commented out, or duplicated (.properties allow that).

Follow the pattern here:
https://github.com/prestosql/presto/blob/d452a6d536e40ed8fb9290711d654e25d6bf07ee/presto-server-rpm/src/main/resources/dist/etc/init.d/presto#L28

Reply:
Fixed